tests/Unit/EmogrifierTest.php

```diff
@@ -758,6 +758,20 @@ public function formattedCssDeclarationDataProvider()
                 '-webkit-text-size-adjust:none;',
                 '-webkit-text-size-adjust: none;'
             ],
+            'one declaration with linefeed in property value' => [
+                'text-shadow:' . static::LF .
+                    '1px 1px 3px #000,' . static::LF .
+                    '1px 1px 1px #000;',
+                'text-shadow: 1px 1px 3px #000,' . static::LF .
+                    '1px 1px 1px #000;',
+            ],
+            'one declaration with Windows line ending in property value' => [
+                'text-shadow:' . "\r\n" .
+                    '1px 1px 3px #000,' . "\r\n" .
+                    '1px 1px 1px #000;',
+                'text-shadow: 1px 1px 3px #000,' . "\r\n" .
+                    '1px 1px 1px #000;',
```

Comment:
You can shorten this a bit by using double quotes for the complete string (without using concatenation and without using the constant). (The rule is to have single quotes unless double quotes are needed.)

Reply:
Done